ob1k-core/src/main/java/com/outbrain/ob1k/client/endpoints/StreamClientEndpoint.java

```diff
@@ -40,9 +41,10 @@
   };
 
   public StreamClientEndpoint(final HttpClient httpClient, final RequestMarshallerRegistry marshallerRegistry,
-                              final Endpoint endpoint, final StreamFilter[] filters) {
+                              final Endpoint endpoint, final StreamFilter[] filters,
+                              final DoubleDispatchStrategy doubleDispatchStrategy) {
 
-    super(httpClient, marshallerRegistry, endpoint);
+    super(httpClient, marshallerRegistry, endpoint, doubleDispatchStrategy);
```

Comment:
maybe it should be more clarified that this feature isn't supported on stream endpoints.